lib/install/examples/stimulus/controllers/hello_controller.js

```diff
@@ -0,0 +1,12 @@
+// Visit The Stimulus Handbook for more details 
+// https://stimulusjs.org/handbook/introduction
```

Comment:
It would be nice to show the HTML this controller expects in the comments here.

Reply:
good idea!